### PR TITLE
Add unload graph/app test cases for lazy mode execution

### LINKED ISSUE #409: Enrich test cases and documentation for lazy mode execution

### PR DESCRIPTION
**Pull request by lidongze0629**
## Related issue number

Fixes #409

### CODE REVIEW

File: python/graphscope/framework/dag_utils.py [hunk 1]
```diff
@@ -659,17 +659,12 @@ def unload_graph(graph):
...
 
     Returns:
         An op to unload the `graph`.
     """
     config = {}
-    if not isinstance(graph, DAGNode):
-        config.update({types_pb2.GRAPH_NAME: utils.s_to_attr(graph.key)})
-        # Dynamic graph doesn't have a vineyard id
-        if hasattr(graph, "vineyard_id"):
-            config[types_pb2.VINEYARD_ID] = utils.i_to_attr(graph.vineyard_id)
```
Comment (siyuan0322) [inline python/graphscope/framework/dag_utils.py:672]: The `unload` doesn't require the `vineyard_id` anymore?

File: python/graphscope/framework/graph.py [hunk 2]
```diff
@@ -95,6 +95,34 @@ def load_from(cls, path, sess, **kwargs):
...
+            >>> nx_g = gs.nx.path_graph(10)
+            >>> gs_g = gs.Graph(nx_g)
+        """
+        if self.session_id != g.session_id:
+            raise RuntimeError(
+                "networkx graph and graphscope graph not in the same session."
+            )
+        if hasattr(g, "_graph"):
+            msg = "graph view can not convert to gs graph"
+            raise TypeError(msg)
```
Comment (siyuan0322) [inline python/graphscope/framework/graph.py:123]: Those two lines can combine into one as the `RuntimeError` above does.

File: python/graphscope/framework/graph.py [hunk 3]
```diff
@@ -154,7 +183,30 @@ def _construct_op_of_empty_graph(self):
...
+        >>> sess = gs.session(mode="lazy")
+        >>> g = sess.g()
+        >>> g1 = g.add_vertices("person.csv","person")
+        >>> sess.run(g1)
+
+        >>> # eager mode
+        >>> import graphscope as gs
+        >>> sess = gs.session(mode="eager")
+        >>> g = sess.g()
+        >>> g1 = g.add_vertices("person.csv","person")
```
Comment (siyuan0322) [inline python/graphscope/framework/graph.py:207]: Better tell user the difference between the two `add_vertices`, i.e. you can print then out.

File: python/graphscope/framework/graph.py [hunk 4]
```diff
@@ -266,6 +321,21 @@ def _resolve_op(self, incoming_data):
             raise RuntimeError("Not supported incoming data.")
 
     def add_vertices(self, vertices, label="_", properties=None, vid_field=0):
+        """Add vertices to the graph, and return a new graph.
+
+        Args:
+            vertices (Union[str, Loader]): Vertex data source.
+            label (str, optional): Vertex label name. Defaults to "_".
+            properties (list[str], optional): List of column names loaded as properties. Defaults to None.
+            vid_field (int, optional): Column index or name used as id field. Defaults to 0.
```
Comment (siyuan0322) [inline python/graphscope/framework/graph.py:330]: this can also be a str, which represents the name of the property.

**Review by siyuan0322 — APPROVED** [review]

**Review by sighingnow — APPROVED** [review]
LGTM.

File: python/graphscope/framework/dag_utils.py (hunk 1 above)
Comment (lidongze0629) [reply python/graphscope/framework/dag_utils.py:672]: move this logic to the coordinator.

**Pull request merged**